docs/template.asciidoc

```diff
@@ -0,0 +1,62 @@
+[[page-title]]
+= Page title
+
+[partintro]
+--
+
+Include a short description. The short description should always include 
+the value prop, answer why users want to use the feature, and how the 
+feature benefits them. 
+
+For easy navigation, include the links to the key sections in a bulleted 
+list. This way, users can click exactly where they want to go instead of 
+scrolling down the page:
+
+* <<task-section-description, Task section title>>
+* <<section2-description, Task section title>>
+* <<section3-description, Task section title>>
+
+Provide a screenshot or video.
+
+TIP: Need inspiration? Here are some good examples:
+* <<snapshot-repositories, Snapshot and Restore>>
+* <<dashboard, Dashboard>>
+* <<canvas, Canvas>>
+
+[float]
+[[concept-section-description]]
+== Concept section title
+
+Include content that further describes the feature and its capabilities.
+
+--
+
+[[task-section-description]]
+== Task section title
+
+Provide an example or walkthrough of how to use the feature. Whenever possible, 
+use the <<add-sample-data, sample data>>.
+
+[float]
+[[section2-description]]
+=== Task section title
+
+To provide steps, use the following:
+
+. Step 1.
+. Step 2.
+. Step 3.
```

Comment:
Maybe show how to indent explanatory text. 

How about some quick guidelines:

match the text to the UI
use bold for clickable items
Click vs. select
avoid using the UI item name.  Click Save and not Click on the Save button.
Use 16:9 ratio for screenshots